### DISCUSSION TITLE
Bug with NaN position

### DISCUSSION THREAD

**Discussion by mikitakuzmichkin** [gilzoide/unity-flex-ui#7] [General]
If there is no flexlayout on the child element, then sometimes when disabling/enabling the object the position becomes NaN. I decide, Nan. I added a disabled flexlayout to the child element

**Comment by mikitakuzmichkin** [comment]
When use inputField with TMP, margins is NaN

**Comment by mikitakuzmichkin** [comment]
i think problem in OnValidate

**Comment by gilzoide** [comment]
This bug was fixed by #16.
